pages/who_we_serve.js

```diff
@@ -30,90 +37,125 @@ const mentorItems = [
+              {!memberCount ? 'over 5,000' : `${memberCount}`} members who are all working towards
+              relevant goals on Slack and in-person meet-ups. Membership is free!
             </p>
 
-            <LinkButton
-              href="https://www.meetup.com/pro/operationcode"
-              analyticsEventLabel="Meetup.com Locations Link"
-              theme="secondary"
+            <div className={classNames(styles.centeredText, styles.topMargin)}>
+              <LinkButton href="/join" theme="secondary">
+                Become A Member
+              </LinkButton>
+            </div>
+          </div>,
+        ]}
+      />
+
+      <Content
+        columns={[
+          <TrackVisibility offset={VISIBILITY_OFFSET}>
+            <ImageCard
+              alt="Two developers collaborting over some code."
+              imageSource={`${s3}redesign/images/paired_programming.jpg`}
+              isImageFirst
             >
-              See Locations
-            </LinkButton>
-          </ImageCard>
-        </TrackVisibility>,
-      ]}
-    />
-
-    <JoinSection />
-  </>
-);
+              <p className={styles.centeredText}>
+                Currently 1,000,000+ <br /> software development jobs available in the United
+                States.
+              </p>
+            </ImageCard>
+          </TrackVisibility>,
+        ]}
+      />
+
+      <Content
+        title="Our Commitment To You"
+        theme="gray"
+        columns={[
+          <p className={styles.justifyAlign}>
+            Whether you are looking to change careers or starting a new one in the tech industry, we
+            are here to help you succeed by providing:
+          </p>,
+          <div className={styles.badgeGroupings}>
+            {mentorItems.map(item => (
+              <Badge
+                key={item.label}
+                icon={item.icon}
+                label={item.label}
+                className={styles.badge}
+              />
+            ))}
+          </div>,
+        ]}
+      />
+
+      <Content
+        columns={[
+          <TrackVisibility offset={VISIBILITY_OFFSET}>
+            <ImageCard
+              alt="Room full of people chatting at an Operation Code meetup in New York City."
+              imageSource={`${s3}redesign/images/chatting_at_meetup.jpg`}
+            >
+              <p className={styles.centeredText}>
+                We have meetup chapters all around the United States!
+              </p>
+
+              <LinkButton
+                href="https://www.meetup.com/pro/operationcode"
+                analyticsEventLabel="Meetup.com Locations Link"
+                theme="secondary"
+              >
+                See Locations
+              </LinkButton>
+            </ImageCard>
+          </TrackVisibility>,
+        ]}
+      />
+
+      <JoinSection />
+    </>
+  );
+}
+
+WhoWeServe.getInitialProps = getMemberCount;
```

Comment:
I'm sorry for not having a lint rule for this, but can you define static properties above the component function? Additionally, can you leave the function unnamed? I'll have to figure out how to lint for that - we made a custom rule to enforce propTypes and defaultProps get defined above the component function.

Like so...
`WhoWeServe.getInitialProps = async () => { /* ...`

Reply:
👍 You got it